roles/shiftstack/molecule/default/converge.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
```

Comment:
If you're consuming CRC, you want to add some more things in here, for instance:
https://github.com/openstack-k8s-operators/ci-framework/blob/7b1ba99b520e3bac5aa73515e321b1e4268fe6e2/roles/openshift_setup/molecule/default/converge.yml#L18-L27